packages-web/datagrid/src/Datagrid.editorPreview.tsx

```diff
@@ -0,0 +1,43 @@
+import { createElement, ReactElement } from "react";
+import { DatagridPreviewProps } from "../typings/DatagridProps";
+
+import "./ui/Datagrid.scss";
+import { Table } from "./components/Table";
+
+export function preview(props: DatagridPreviewProps): ReactElement {
+    const data = Array.from({ length: props.pageSize ?? 5 }).map(() => ({
+        id: "" as any
+    }));
+
+    return (
+        <Table
+            className={props.class}
```

Comment:
don't we loose an inline styles? Also in actual grid component as well

Reply:
woww, good catch

Reply:
Done :)